app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/TimerActivity.java

```diff
@@ -63,7 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
             builder.setPositiveButton("OK", new DialogInterface.OnClickListener() {
                 public void onClick(DialogInterface dialog, int id) {
-                    new CountDownTimer(tracosCapsula1.get(pos)*3*1000, 1000) {
+                    new CountDownTimer(bars1 * secondsPerBar, 1000) {
```

Comment:
If you are going to replicate this logic you could maybe create an inner class with it.